### ISSUE TITLE
Add support for native virtual psvita keyboard

### ISSUE THREAD

**Issue by AorsiniYT** [MetalfaceScout/vita-moonlight-motion#2]
Will it be possible to adapt a virtual keyboard as is done in this repository? https://github.com/GrapheneCt/vita-moonlight
Unfortunately I couldn't compile it but I did try the keyboard in an old version of Geforce Experience and the keyboard works very well to write and not depend on the Steam keyboard

**Comment by MetalfaceScout** [comment]
Yes, this would be possible. I'll add it as a feature after native gyro support.

**Comment by AorsiniYT** [comment]
Thank you very much, I tried your new version and the gyroscope works very well, you could add shortcuts with the PS button on the PSVita, I have seen in the vitaoxivita app that when you enter control mode the PS button on the PSVita does not take you out of the app, in Moonlight you could add shortcuts like pressing (PS + X) and it could perform some function, it is an example, so you could easily add the absolute touchscreen, although these are just ideas.
